safe-global/web-core
# fix: wrap relative links in next/link

## What it solves

Resolves #794

## How this PR fixes it
Wraps relative links in `next/link` avoiding a page reload

## How to test it
1. Connect with an account that does NOT own a Safe
2. Navigate to a shared Transaction Builder link
3. Click "Create new Safe"
4. The navigation to the Create Safe screen should be smooth and not lose your wallet connection

## Analytics changes
None

## Screenshots
<img width="890" alt="Screenshot 2022-10-06 at 11 35 40" src="https://user-images.githubusercontent.com/32431609/194279523-efc2ea08-293e-4eb3-ad87-e5ccd086b4d3.png">
